.gitmodules

```diff
@@ -1,9 +1,12 @@
 [submodule "uSockets"]
-	path = uSockets
-	url = https://github.com/uNetworking/uSockets.git
+path = uSockets
+url = https://github.com/Jarred-Sumner/uSockets
```

Comment:
You probably don't want to commit this

Reply:
you can checkout your variant until uSockets is merged